src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -191,6 +188,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.LookupAccountNameW.cs">
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.SHGetKnownFolderPath.cs">
```

Comment:
I see ... I thought you mean different ones. The two with the same label are fine to merge.